systemvm/debian/etc/systemd/system/cloud-postinit.service

```diff
@@ -1,7 +1,7 @@
 [Unit]
 Description=CloudStack post-patching init script
 After=cloud-early-config.service network.target local-fs.target
-Before=ssh.service
+Before=ssh.service apache2.service
```

Comment:
This may require a new systemvm template, as dynamic patching may not change the process dependency graph. (needs testing/verification).

Reply:
Does this mean that 4.11.1 will require another system VM?

Reply:
It depends, we can publish a new systemvm if needed. Those facing issues may use the new updated template. There seems to be a new (minor?) linux security patch as well that could get included as well.

Reply:
I checked the code again, the cloud-early-config service runs first that patches the vr/systemvm using the new systemvm.iso (that will contain this change) and does a systemctl daemon-reload before cloud-postinit could run. In that case, it's possible that a new systemvmtemplate is not needed. We'll need to verify this manually, or via the systemd flame graph (@resmo can you share/attach the `systemd-analyse plot > svg`? or some other log/output)

Reply:
@resmo post-merging this PR I found that sometimes isolated network VRs get stuck when starting due to unable to restart apache2 via postinit script. Can you advise why apache2 need to be started after postinit script runs?

Reply:
We see systemd errors unable to start apache on the first boot, It seemed to be related to missing network setup but after postinit script ran, apache were "restarted" and apache was accessible again.

It is really hard to predict the dependency tree in this scripting construct...

Reply:
You can see the dependency graph for systemd, via systemd-analyze (plot it or analyze the critical-chain etc, refer docs for more info). Apache2 is disabled by default, so during first boot no errors should be seen. Can you share when you saw those errors, did that cause any service related failure for you, also where you saw these errors (in console or logs?). The cloud-postinst script will handle any failure and start apache2 is required, any initial error/warning may be ignored.

Reply:
I meant the dependency caused in the scripts not systemd.

However, I am ok to revert this if it causes serious issues.